[tests] Fix flaky tests

I have observed that CI build for `Python==3.7 | django~=3.0.0` fails sometimes abruptly. 

One of such occurrence is https://github.com/openwisp/openwisp-controller/pull/421/checks?check_run_id=2622952481

Let's use this issue to report such flaky tests and fix them all at once.

Build fail: https://github.com/openwisp/openwisp-controller/runs/2719061481

```
======================================================================
FAIL: test_ca_queryset (openwisp2.sample_pki.tests.TestAdmin) [ca2]
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/site-packages/openwisp_users/tests/utils.py", line 99, in _test_multitenant_admin
    msg_prefix='[operator not-contains]',
  File "/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/site-packages/django/test/testcases.py", line 465, in assertNotContains
    self.assertEqual(real_count, 0, msg_prefix + "Response should not contain %s" % text_repr)
AssertionError: 1 != 0 : [operator not-contains]: Response should not contain 'ca2'
```

--- a/openwisp_controller/pki/tests/test_admin.py
+++ b/openwisp_controller/pki/tests/test_admin.py
@@ -20,8 +20,8 @@ class TestAdmin(TestPkiMixin, TestAdminMixin, TestOrganizationMixin, TestCase):
         inactive = self._create_org(name='inactive-org', is_active=False)
         operator = self._create_operator(organizations=[org1, inactive])
         administrator = self._create_administrator(organizations=[org1, inactive])
-        ca1 = self._create_ca(name='ca1', organization=org1)
-        ca2 = self._create_ca(name='ca2', organization=org2)
+        ca1 = self._create_ca(name='Org1 CA', organization=org1)
+        ca2 = self._create_ca(name='Org2 CA', organization=org2)
         ca_shared = self._create_ca(name='ca-shared', organization=None)
         ca_inactive = self._create_ca(name='ca-inactive', organization=inactive)
         data = dict(
@@ -36,8 +36,8 @@ class TestAdmin(TestPkiMixin, TestAdminMixin, TestOrganizationMixin, TestCase):
             administrator=administrator,
         )
         if cert:
-            cert1 = self._create_cert(name='cert1', ca=ca1, organization=org1)
-            cert2 = self._create_cert(name='cert2', ca=ca2, organization=org2)
+            cert1 = self._create_cert(name='Org1 Cert', ca=ca1, organization=org1)
+            cert2 = self._create_cert(name='Org2 Cert', ca=ca2, organization=org2)
             cert_shared = self._create_cert(
                 name='cert-shared', ca=ca_shared, organization=None
             )

